services/slam/slam.go

```diff
@@ -87,3 +91,21 @@ func GetInternalStateFull(ctx context.Context, slamSvc Service) ([]byte, error)
 	}
 	return HelperConcatenateChunksToFull(callback)
 }
+
+// GetLimits returns the bounds of the slam map as a list of referenceframe.Limits.
+func GetLimits(ctx context.Context, svc Service) ([]referenceframe.Limit, error) {
+	data, err := GetPointCloudMapFull(ctx, svc)
+	if err != nil {
+		return nil, err
+	}
+	dims, err := pointcloud.GetPCDMetaData(bytes.NewReader(data))
+	if err != nil {
+		return nil, err
+	}
+
+	return []referenceframe.Limit{
+		{Min: dims.MinX, Max: dims.MaxX},
+		{Min: dims.MinY, Max: dims.MaxY},
+		{Min: -2 * math.Pi, Max: 2 * math.Pi},
```

Comment:
Probably we do not need this last bit

Reply:
Could we leave it in? I imagine it would be useful when we are doing akerman steering

Reply:
and right now the last sublist would just be overlooked right?

Reply:
We shouldn't pass meaningless values. It's not useful for ackerman steering, because the inputs (and therefore what the limits represent) are wholly different for tp-space frames.